docs/psa/index.md

```diff
@@ -21,24 +21,27 @@ PSA includes functional specifications describing a hardware abstraction layer c
 
 ### Accelerator driver interface
 
-The accelerator interface lets you drivers for cryptographic accelerators into an implementation of the PSA Cryptography API. Cryptographic accelerators perform cryptographic operations with keys in clear text.
+The cryptographic accelerator driver interface lets you write drivers for hardware that performs cryptographic operations with keys in clear text.
+You can plug such drivers into any implementation of the PSA Cryptography API.
 
 For more information, see [PSA cryptography accelerator driver interface](accel/).
 
 ### Secure element driver interface
 
-The accelerator interface lets you drivers for external cryptoprocessors into an implementation of the PSA Cryptography API. External cryptoprocessors such as secure elements and smart cards perform cryptographic operations with keys accessed via opaque handles.
+The secure element driver interface lets you write drivers for external cryptoprocessors such as secure elements (SE), smart cards and hardware security modules (HSM) that perform operations on keys that never leave the external processor and are accessed only through opaque handles.
+You can plug such drivers into any implementation of the PSA Cryptography API.
```

Comment:
Do you want another newline above here like in the other sections?

Reply:
I don't understand. A newline where? Are you talking about source formatting or rendering?

Reply:
Source format.

Here, we have the following in source.

```
The secure element driver interface lets you write drivers for external cryptoprocessors such as secure elements (SE), smart cards and hardware security modules (HSM) that perform operations on keys that never leave the external processor and are accessed only through opaque handles.
You can plug such drivers into any implementation of the PSA Cryptography API.
```

In other similar places, we have:
![Screenshot at 2019-10-16 09-29-23](https://user-images.githubusercontent.com/410816/66901891-7deaa100-eff7-11e9-9bae-69b5c3123952.png)


However, I now see I was fooled by GitHub adding the vertical whitespace. There is no extra newline present.